api/tests/unit/domain/services/answer-service_test.js

```diff
@@ -0,0 +1,85 @@
+const { describe, it, expect } = require('../../../test-helper');
+const certificationService = require('../../../../lib/domain/services/answer-service');
+
+const Bookshelf = require('../../../../lib/infrastructure/bookshelf');
+const Answers = require('../../../../lib/domain/models/data/answer');
+
+describe('Unit | Domain | Services | answer-service', function() {
+
+  describe('#getAnswersSuccessRate', () => {
+
+    const AnswersCollection = Bookshelf.Collection.extend({
+      model: Answers
+    });
+
+    it('should exists', () => {
+      expect(certificationService).to.have.property('getAnswersSuccessRate')
+        .and.to.be.a('function');
+    });
```

Comment:
je l'aime bien 👍 mais c'est un test squelette

Reply:
just crushed it 💀